.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,72 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+	"name": "C# (.NET)",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/dotnet:1-9.0",
+	"features": {
+		"ghcr.io/devcontainers/features/azure-cli:1": {},
+		"ghcr.io/azure/azure-dev/azd:0": {},
+		"ghcr.io/devcontainers/features/docker-in-docker": {},
+		"ghcr.io/devcontainers/features/dotnet": {
+			"additionalVersions": [
+				"8.0.403"
+			]
+		}
+	},
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	"forwardPorts": [
+		15887, // playground/waitfor AppHost
+		5180, // playground/waitfor ApiService
+		7024, // playground/waitfor Frontend
+		15551 // playground/waitfor PGAdmin
```

Comment:
We can let devcontainers automatically map ports, however in the case of .NET Aspire we open so many ports that it can be overwhelming. Current approach is to explicitly map ports here. We probably should go through and make sure we have no overlapping ports which will probably help in other areas too around cookies etc.